projects/ui/src/Apis/api-types.ts

```diff
@@ -171,9 +171,25 @@ export type OauthCredential = {
   idpClientName: string;
 };
 
+// This list of units is used both for the type and for the dropdown in the UI.
+const rateLimitUnits = [
+  "UNKNOWN",
+  "SECOND",
+  "MINUTE",
+  "HOUR",
+  "DAY",
+  "MONTH",
+  "YEAR",
+] as const; // The 'as const' tells TypeScript to treat these as literal types
+export const rateLimitUnitOptions = rateLimitUnits.map((unit) => ({
+  value: unit,
+  label: unit,
+}));
```

Comment:
I added this here so it just calculates the options once and not on each render (which happens when it's in the JSX).

Reply:
Ahh smart, I see, since those units are static no need for it to do on-render 👍🏼